docs/blog/newsletter-october-2019.rst

```diff
@@ -0,0 +1,116 @@
+.. post:: October 04, 2019
+   :tags: newsletter
+
+########################################
+Write the Docs Newsletter – October 2019
+########################################
+
+Hi, everyone, and welcome back to the newsletter after our summer break!
+
+Top news this month is our salary survey. It can be tough to work out what "normal" pay is, so Write the Docs is carrying out a `salary survey </surveys/salary-survey-sep-2019/>`__ to try to get some answers. The survey is open until 31st October, and we'll be publishing the anonymized data for free. Help the community and `fill it out here <https://www.surveymonkey.com/r/wtd-salary-2019>`__!
+
+On the events side, I thoroughly enjoyed meeting so many of you at the Prague conference! If you missed it, don't despair, because `all the talks are now up on YouTube <https://www.youtube.com/playlist?list=PLZAeFn6dfHpkpYchP1iFnQnc7i-2xJd0I>`__ for your perusal. Plus there's one last conference to look forward to this year: we've announced the `speakers and schedule </conf/australia/2019/news/announcing-speakers/>`__ for Write the Docs Australia, coming up in November. 
```

Comment:
```suggestion
On the events side, I thoroughly enjoyed meeting so many of you at the Prague conference! If you missed it, don't despair, because `all the talks are now up on YouTube <https://www.youtube.com/playlist?list=PLZAeFn6dfHpkpYchP1iFnQnc7i-2xJd0I>`__ for your enjoyment. Plus there's one last conference to look forward to this year: we've announced the `speakers and schedule </conf/australia/2019/news/announcing-speakers/>`__ for Write the Docs Australia, coming up in November. 
```

Reply:
Feels like a 'slightly' odd use of perusal, but I'm happy to be overrruled.